dataset-snippets/measurement_by_age_and_gender.plotnine

```diff
@@ -0,0 +1,33 @@
+# Use snippet 'measurement_by_age_and_gender' to plot joined demographics and measurements dataframes.
+# This plot assumes 'measurement_df' was created using snippet 'Basic operations -> join_dataframes' to
+# join together demographics and measurements dataframes.
+# See also https://plotnine.readthedocs.io/en/stable/
+
+
+# There could be many different measurements in the dataframe. By default, plot the first one.
+measurement_to_plot = measurement_df.STANDARD_CONCEPT_NAME.unique()[0]
+
+# Create a derived variable for age group.
+measurement_df['age'] = (pd.to_datetime('today').year
+                                     - measurement_df.DATE_OF_BIRTH.map(lambda x: x.year))
+measurement_df['age_group'] = pd.cut(measurement_df['age'],
+                                                 [-np.inf, 34.5, 49.5, 64.5, np.inf],
+                                                 labels=["<35", "35-49", "50-64", "65+"])
+
+# meas_filter is a column of True and False
+meas_filter = ((measurement_df.STANDARD_CONCEPT_NAME == measurement_to_plot)
```

Comment:
I get a few charts just titled `nan` when trying this on body height, perhaps some empty values are not being filtered?

Reply:
Added the filters for nulls.